mapie/classification.py

```diff
@@ -805,59 +735,36 @@ def _get_last_included_proba(
             with the RAPS method, the index of the last included score
             and the value of the last included score.
         """
-        index_sorted = np.flip(
-            np.argsort(y_pred_proba, axis=1), axis=1
-        )
+        index_sorted = np.flip(np.argsort(y_pred_proba, axis=1), axis=1)
         # sort probabilities by decreasing order
-        y_pred_proba_sorted = np.take_along_axis(
-            y_pred_proba, index_sorted, axis=1
-        )
+        y_pred_proba_sorted = np.take_along_axis(y_pred_proba, index_sorted, axis=1)
         # get sorted cumulated score
-        y_pred_proba_sorted_cumsum = np.cumsum(
-            y_pred_proba_sorted, axis=1
-        )
+        y_pred_proba_sorted_cumsum = np.cumsum(y_pred_proba_sorted, axis=1)
 
         if self.method == "raps":
             y_pred_proba_sorted_cumsum += lambda_ * np.maximum(
-                0,
-                np.cumsum(
-                    np.ones(y_pred_proba_sorted_cumsum.shape),
-                    axis=1
-                ) - k_star
+                0, np.cumsum(np.ones(y_pred_proba_sorted_cumsum.shape), axis=1) - k_star
             )
         # get cumulated score at their original position
         y_pred_proba_cumsum = np.take_along_axis(
-            y_pred_proba_sorted_cumsum,
-            np.argsort(index_sorted, axis=1),
-            axis=1
+            y_pred_proba_sorted_cumsum, np.argsort(index_sorted, axis=1), axis=1
         )
         # get index of the last included label
         y_pred_index_last = self._get_last_index_included(
-            y_pred_proba_cumsum,
-            thresholds,
-            include_last_label
+            y_pred_proba_cumsum, thresholds, include_last_label
         )
         # get the probability of the last included label
-        y_pred_proba_last = np.take_along_axis(
-            y_pred_proba,
-            y_pred_index_last,
-            axis=1
-        )
+        y_pred_proba_last = np.take_along_axis(y_pred_proba, y_pred_index_last, axis=1)
 
-        zeros_scores_proba_last = (y_pred_proba_last <= EPSILON)
+        zeros_scores_proba_last = y_pred_proba_last <= EPSILON
 
         # If the last included proba is zero, change it to the
         # smallest non-zero value to avoid inluding them in the
         # prediction sets.
         if np.sum(zeros_scores_proba_last) > 0:
             y_pred_proba_last[zeros_scores_proba_last] = np.expand_dims(
-                np.min(
-                    np.ma.masked_less(
-                        y_pred_proba,
-                        EPSILON
-                    ).filled(fill_value=np.inf),
-                    axis=1
-                ), axis=1
+                np.min(np.ma.masked_less(y_pred_proba, EPSILON).filled(fill_value=np.inf), axis=1),
```

Comment:
An example where I think it's less readable after black : here you have 4 numpy functions in 3 lines (expand_dims, min, masked_less, filled), quite hard to understand

Reply:
In this case, I should have added pre-commit first, then thought about black.